sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorCodeSnippet.java

```diff
@@ -2,25 +2,51 @@
 // Licensed under the MIT License.
 package com.azure.cosmos;
 
+import com.fasterxml.jackson.databind.JsonNode;
+
 /**
  * Code snippets for {@link ChangeFeedProcessor}
  */
 public class ChangeFeedProcessorCodeSnippet {
 
     public void changeFeedProcessorBuilderCodeSnippet() {
-        String hostName = null;
-        CosmosAsyncContainer feedContainer = null;
-        CosmosAsyncContainer leaseContainer = null;
+        String hostName = "test-host-name";
```

Comment:
If we reference this in the docs, we should make sure that we capture that this string must be unique like suffix this with a random generated string

Reply:
This doesn't get referenced in the docs at all.. This was just to fix the javadocs warnings.

Reply:
Docs only show what is between BEGIN and END tags